docs/image/bounding_box.rst

```diff
@@ -29,19 +29,17 @@ Let's say you have a large image, but are only interested in a small box (a rect
    np.sum(full_array)
    np.sum(sub_array)
 
-Numpy supports working with such boxes efficiently through the concept of 
+Numpy supports working with such boxes efficiently through the concept of
 `slicing <http://scipy-lectures.github.io/intro/numpy/array_object.html#indexing-and-slicing>`__
-and 
+and
 `views <http://scipy-lectures.github.io/intro/numpy/array_object.html#copies-and-views>`__.
 
 By slicing ``[1000:1010, 2000:2020]`` we created a view (not a copy) ``sub_array`` into the ``full_array``.
-On my machine making a measurement on ``sub_array`` is about 1000 times as fast as for ``full_array``:
-
-.. code-block:: python
```

Comment:
This warning and build fail showed up in Gammapy master today:
https://travis-ci.org/gammapy/gammapy/jobs/106266631#L2024

It's fixed in this PR, so I'll not fix it in master.
@bsipocz - OK to merge this PR later today and leave extra stuff like Sherpa dev version or extra mac builds to future PRs?
Or should I fix this in master and then you rebase this PR?